crossbeam-epoch/src/atomic.rs

```diff
@@ -487,7 +506,14 @@ impl<T> Atomic<T> {
     /// }
     /// ```
     pub unsafe fn into_owned(self) -> Owned<T> {
-        Owned::from_usize(self.data.into_inner())
+        #[cfg(loom)]
+        {
+            Owned::from_usize(self.data.unsync_load())
```

Comment:
I'd like loom to support into_inner.  I believe matching the APIs with std is one of the design goals of loom, right?

Reply:
Unfortunately there are still some holes, and `into_inner` is one of them: https://github.com/tokio-rs/loom/issues/117